backend/app/views/spree/admin/orders/customer_details/edit.html.erb

```diff
@@ -1,11 +1,10 @@
 <%= render :partial => 'spree/admin/shared/order_tabs', :locals => { :current => 'Customer Details' } %>
 
 <% content_for :page_title do %>
-  <i class="fa fa-arrow-right"></i> <%= Spree.t(:customer_details) %>
+  &#x2f; <%= Spree.t(:customer_details) %>
 <% end %>
 
 <% content_for :page_actions do %>
```

Comment:
Just checked this out.  Without adding a &nbsp; or something to the block the `<ul class='page-actions'>` element doesn't show up on the page.  I'm thinking we'd probably like to make sure that we don't lose those in case people are hooking into them.  I will make sure I change that.